truss-chains/examples/baseten_llm/trt_llama.py

```diff
@@ -0,0 +1,46 @@
+from typing import AsyncIterator
+
+from truss.base import trt_llm_config, truss_config
+
+import truss_chains as chains
+
+
+class Llama7BChainlet(chains.EngineBuilderLLMChainlet):
+    remote_config = chains.RemoteConfig(
+        compute=chains.Compute(gpu=truss_config.Accelerator.H100),
+        assets=chains.Assets(secret_keys=["hf_access_token"]),
+    )
+    engine_builder_config = truss_config.TRTLLMConfiguration(
+        build=trt_llm_config.TrussTRTLLMBuildConfiguration(
```

Comment:
I don't love the naming of 
```
TrussTRTLLMBuildConfiguration
```

Very heavy, I know this is a remnant of how MP implemented this for Engine Builder but maybe there are better namingconventions we can use here.